Service updates

cuete opened the pull request:
Adding latest API changes from Senegal Dashboard:
- API controllers
- Helper methods
- Schema updates
- Test suite

- Removed checked-in test data

TODO: UI must be updated to use the schema changes

Commit:
gitignore

Commit:
Updating service API code and tests

dkong-bmgf reviewed (commented):
Pls add the followings:
1) add script to convert from old data format to new format. 
2) convert data (*.csv) to new format

Commit:
Adding test data and converter script

dkong-bmgf commented on service/Dockerfile:
```diff
@@ -7,28 +7,17 @@ ADD pip.conf /root/.pip/pip.conf
 ENV PYTHONPATH=/app:${PYTHONPATH}
 ENV PATH=/app:${PATH}
 
-RUN mkdir -p /app/service/helpers
```
we still need the scripts to download geojson for licensing reason.  Pls revert the whole file.

cuete replied:
File reverted.

dkong-bmgf commented on service/data/shapefiles/Senegal__l2__1.shp.pickle_old:
Pls remove the pickle files as they will be downloaded by users.  Check follow README.md

cuete replied:
Pickle files removed

dkong-bmgf commented on service/data/layer_data.json:
Pls remove data

dkong-bmgf commented on service/data/layer_data.json:
Pls remove data

cuete replied:
Data files updated with dummy data.

dkong-bmgf commented on service/Dockerfile.local:
Pls revert this file  as we still need steps to downloadGeoJson.js.  Afterwards, add the steps to run test.  Pls deploy locally and check if api works

cuete replied:
Reverted and tested.

dkong-bmgf reviewed (commented):
Pls review my comments. Thanks.

Commit:
Data file updates

Commit:
Changing back docker.local

cuete commented:
- Replaced data with dummy data:
    - layer_data
    - indicator csv files
    - events.csv
- Added converter documentation to Readme.md
- Removed shapefiles
- Reverted and updated the service readme.md
- Reverted and updated dockerfiles in service

*Service tested in docker and debugger. 
*Unit tests passed.
* UI needs updates to use the new API endpoints

dkong-bmgf committed:
Updated FE with these features:
- month slider
- dynamic year slider
- support district selection (map click) on comparison map
- support for stacked column chart
- support for multiple languages
- improved page routing

dkong-bmgf reviewed (approved):
Looks good
